server/request/authenticateRequest.js

```diff
@@ -12,8 +12,8 @@ let scopesBySite;
 
 const authenticateRequest = async ({ site, route, options, ...requestOptions }) => {
   const requestUrlsBySite = {
-    graph: `${options.graphApiUrl}v1.0/security/`,
-    defender: `${options.microsoft365ApiUrl}api/`
+    graph: `${options.graphApiUrl}/v1.0/security/`,
```

Comment:
@penwoodjon Will this API work if there is a double slash?  I know in some previous instances we've put in a check for this possibility.  For example, if someone put in `https://myapi/` for the option and then the url becomes `https://myapi//v1.0`

Reply:
Thanks for double checking on this! This possibility should be accounted for in the validateOptions 👍 
https://github.com/polarityio/microsoft-defender/blob/main/server/userOptions/utils.js#L49-L54